test/test.getch.ts

```diff
@@ -42,16 +41,14 @@ const url = 'https://example.com';
 
 describe('🦖 option validation', () => {
   it('should throw an error if a url is not provided', () => {
-    assertRejects(request({}), /URL is required/);
+    assert.rejects(request({}), /URL is required/);
```

Comment:
should it be `await assert.rejects(...)`?

Reply:
Hey great catch!